src/ssl/helper.cc

```diff
@@ -72,7 +72,7 @@ operator <<(std::ostream &os, const Ssl::GeneratorRequest &gr)
 /// pending Ssl::Helper requests (to all certificate generator helpers combined)
 static Ssl::GeneratorRequests TheGeneratorRequests;
 
-helper::Pointer Ssl::Helper::ssl_crtd = nullptr;
+Helper::Client::Pointer Ssl::Helper::ssl_crtd = nullptr;
```

Comment:
`Pointer` do not need explicit nullptr assignment.
```suggestion
Helper::Client::Pointer Ssl::Helper::ssl_crtd;
```

Reply:
Done.